tests/improved-sanity-test/main.yml

```diff
@@ -324,10 +325,11 @@
         - redhat_unsubscribe
 
     # Verify that rpm-ostree can be run as non-root or sudo
-    - role: rpm_ostree_status
-      become: no
+    - role: command_privileged_verify
+      cmd: rpm-ostree status
+      user: bin
       tags:
-        - rpm_ostree_status
+        - command_privileged_verify
         - cloud_image
```

Comment:
On line 333, I don't think we need the second `cloud_image` tag.  Since no upgrade is taking place, we would just be running the same test twice in the cloud image case.  (This isn't technically part of this review, but I just noticed it now).